### ISSUE TITLE
Weather not working with Orion 2.3.0

### ISSUE THREAD

**Issue by artisonus** [FIWARE/tutorials.Context-Providers#17]
I'm following the tutorial, but I'm unable to obtain weather live data (OpenWeatherMap api) related to a store. The live weather health check works fine.

I've registered the context-provider with the URL http://context-provider:3000/weather/weatherConditions, but when I obtain the store information (http://{{orion}}/v2/entities/urn:ngsi-ld:Store:001), the weather information is not showing, only the data in database appears. When retrieving live data from http://{{context-provider}}/weather/weatherConditions/op/query with the payload from the tutorial, I get a bad request with the message "bad query", and the URL to the OpenWeatherMap show q=undefined. The logs also show "tutorial:proxy Making a OpenWeatherMap API request: undefined".

If I change the Orion to version 2.2.0 and register the context provider with legacyForwarding equal to true, all works fine.

How can I make the tutorial work with Orion 2.3.0?

Thanks

**Comment by jason-fox** [comment]
There was a bug in the NGSI-v2 proxy code for the Weather API - a fix has been applied here: [ef9c08b](https://github.com/FIWARE/tutorials.Step-by-Step/commit/ef9c08b1cab6618eb3814dd59f099b857b87ac79)

If you run `./services create`, you should be able to get the latest tutorial proxy which offers NGSI-v1, NGSI-v2 and NGSI-LD endpoints.

```console
curl -iX POST \                                               
  'http://localhost:3000/weather/weatherConditions/op/query' \
  -H 'Content-Type: application/json' \
  -d '{
    "entities": [
        {
            "type": "Store",
            "isPattern": "false",
            "id": "urn:ngsi-ld:Store:001"
        }
    ],
    "attrs": [
        "temperature",
        "relativeHumidity"
    ]
}'
```

Should now return:

```json
[
   {
      "id":"urn:ngsi-ld:Store:001",
      "type":"Store",
      "temperature":{"type":"Number","value":284.02},
      "relativeHumidity":{"type":"Number","value":50}
   }
]
```

**Comment by jason-fox** [comment]
Bug fixed in NGSI-v2 proxy [code](https://github.com/FIWARE/tutorials.Step-by-Step/commit/ef9c08b1cab6618eb3814dd59f099b857b87ac79). Closing
